pkg/message/object_meta_message.go

```diff
@@ -7,19 +7,19 @@ import (
 	"github.com/yezzey-gp/yproxy/pkg/storage"
 )
 
-type ObjectMetaMessage struct {
-	Content []*storage.S3ObjectMeta
+type FilesInfo struct {
```

Comment:
Why? Object Meta Message fits better. If you have local storage, you have object stored there, same for S3